test/parallel/test-vm-new-script-new-context.js

```diff
@@ -38,14 +38,15 @@ console.error('thrown error');
 script = new Script('throw new Error(\'test\');');
 assert.throws(function() {
   script.runInNewContext();
-}, /test/);
+}, /^Error: test$/);
 
 
 console.error('undefined reference');
 script = new Script('foo.bar = 5;');
 assert.throws(function() {
+  script.create;
```

Comment:
I just re-checked it with https://github.com/nodejs/node/blob/master/test/parallel/test-vm-new-script-new-context.js 
It's a sure-shot mistake.